pkg/process/manager.go

```diff
@@ -603,12 +603,34 @@ func (m *Manager) Reconnect(ctx context.Context) ([]*daemon.Daemon, error) {
 		m.daemonStates.RecoverDaemonState(d)
 
 		d.Once = &sync.Once{}
```

Comment:
Why do we need to prepare to initialize the client here?

Reply:
Because method `CheckStatus` is initializing HTTP client when reconnecting to daemons